components/Autocomplete/index.js

```diff
@@ -124,7 +124,7 @@ class Autocomplete extends Component {
     if (resetOnSelect) {
       this.setState({inputValue: ""});
     } else {
-      const capitalizedValue = capitalizeFirstLetter(valueExtractor(value));
+      const capitalizedValue = this.props.firstLetterCapital === true ? capitalizeFirstLetter(valueExtractor(value)) : valueExtractor(value);
```

Comment:
Would be cleaning to just do `this.props.firstLetterCapital` for the conditional

Reply:
You're right. I'm implementing your suggestion, thanks.